scania-digital-design-system/tegel
# docs(prefix change): announce prefix change

**Describe pull-request**  
 - Adds Announcements section.
 - Annoucements use our component and style - no MDX
 - Redid the landing page to use our components.
 - Excluded stories files from build

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1698

**How to test**  
1. Go to storybook
2. Check in Tegel Design System and Announcements
